routes/media.go

```diff
@@ -296,3 +297,93 @@ func (fes *APIServer) GetFullTikTokURL(ww http.ResponseWriter, req *http.Request
 		return
 	}
 }
+
+func (fes *APIServer) UploadVideo(ww http.ResponseWriter, req *http.Request) {
+	if fes.Config.CloudflareStreamToken == "" || fes.Config.CloudflareAccountId == "" {
+		_AddBadRequestError(ww, fmt.Sprintf("UploadVideo: This node is not configured to support video uploads"))
+		return
+	}
+	url := fmt.Sprintf("https://api.cloudflare.com/client/v4/accounts/%v/stream?direct_user=true", fes.Config.CloudflareAccountId)
+	client := &http.Client{}
+
+	request, err := http.NewRequest("POST", url, nil)
+	request.Header.Add("Authorization", fmt.Sprintf("Bearer %v", fes.Config.CloudflareStreamToken))
+	request.Header.Add("Tus-Resumable", "1.0.0")
+	request.Header.Add("Upload-Length", req.Header.Get("Upload-Length"))
```

Comment:
Is there any check on video upload size? Do we need one?

Reply:
Yea we can check the Upload-Length here as well in the frontend. What do you think would be a reasonable size for video uploads? Youtube supports up to 256GB (which seems excessive) I was thinking something in the 16GB to 64GB range as a limit.

Reply:
Added a 4GB limit here and on the frontend. We can make it configurable later.